### DISCUSSION TITLE
MRTK3: Slider graphic styles

### DISCUSSION THREAD

**Discussion by camnewnham** [microsoft/MixedRealityToolkit-Unity#11065] [General]
## Describe the problem

@Zee2 Continuing from the dropdown discussion [over here](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11063), I've been tinkering with how to get sliders to fit in better with the graphic theme of buttons and toggles on canvas-based menus. This one uses the same `CanvasSlider` prefab and adds a frontplate.

I quite like how this one is working in the vein of UI interactable being variants on `ActionButton` . Existing slider at the bottom for comparison.

![slider](https://user-images.githubusercontent.com/19278856/193140413-37d3bce6-e0da-4aa5-8d57-37d956fae480.gif)

## Describe the solution you'd like

If there's interest, I'd like to work it up slightly and get some assistance with the UX/animations to make it as clear and flexible as possible. At the moment it's perhaps less obvious what it is (for example, the grab handle doesn't highlight or animate)

**Comment by camnewnham** [comment]
Aside: should this belong in Issues or Discussions?

**Reply by Zee2** [reply]
Moved to Discussions!

**Comment by Zee2** [comment]
Hey @camnewnham, these UX controls are built to the specification of our internal Mixed Reality Design Language which specifies how these sliders look and behave. The existing slider design will remain as it is, as it does correctly implement the slider from our internal design system (which is closely aligned with, but not exactly, the same as the Windows/WinUI design system.) See Windows sliders as an example:

<img width="482" alt="image" src="https://user-images.githubusercontent.com/5544935/193144019-16746b3f-48aa-4604-a926-2e1bf8248200.png">

 There is another type of slider like you've shown here that our MRDL does specify, but it's not the main/primary type of Slider, so that's why we haven't implemented it yet.

By the way, the grab handle for the MRDL slider should be animating based on hover and select actions! Please file a bug (or check your configuration) if it's not properly animating!
